### PR TITLE
Adding a Vagrantfile for spinning up a development VM

### PR DESCRIPTION
**Pull request by idosinger98**
PR resolves Issue #2 
 
Vagrant is a tool for using virtual machines to share development environments.

Also making changes to the `.gitignore` file to ignore any data that Vagrant might create.

### CODE REVIEW

**Review by TalReinfeld — COMMENTED** [review]
LGTM

**Review by guybeckenstein — COMMENTED** [review]
LGTM

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  # config.vm.network "forwarded_port", guest: 80, host: 8080
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine and only allow access
+  # via 127.0.0.1 to disable public access
+  # config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+  config.vm.network(
+	"forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
+  )
+  config.vm.provision "shell", path: "setup.sh", privileged: false
```
Comment (Yarboa) [inline Vagrantfile:40]: Is setup.sh included in PR
In case some one else will do the work open an Issue
and ask to update this line in vagrant file

Comment (patrisiakaplun) [reply Vagrantfile:40]: @Yarboa 
Hi, I am responsible for adding the setup.sh file.
It is included in another pull request #6

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**